shumai/tensor/tensor.ts

```diff
@@ -306,6 +322,10 @@ export class Tensor {
     return Number(fl._ndim.native(this.ptr))
   }
 
+  get dType() {
```

Comment:
nit: can we call these `dtype` and `astype` (just to be consistent with some other ML stuff/Flashlight, I know it's not exactly JS standard)?

Reply:
Fixed in latest update!